DigraphyApi/Controllers/OrderController.cs

```diff
@@ -19,4 +19,14 @@ public async Task<ActionResult<List<OrderDto>>> GetOrders([FromQuery] int? colle
         var ordersDto = ordersResult.Data;
         return Ok(ordersDto);
     }
+    
+    [HttpGet("{orderId}")]
+    [ProducesResponseType(200, Type = typeof(IEnumerable<Title>))]
+    [ProducesResponseType(400)]
+    public async Task<ActionResult<List<TitleDto>>> GetOrder(int orderId)
```

Comment:
I think this should return a `TitleOrderDto` perhaps, so that it includes the `Order` information too?